### ISSUE TITLE
ctest --tests-information produces error

### ISSUE THREAD

**Issue by pcorbineau** [orjangj/neotest-ctest#25]
on my setup,
Windows : 11
cmake : 3.28
neovim : 0.10.0

I found that line : 
`    local filter = string.format("-I 0,0,0,%s", table.concat(runnable_tests, ","))`

produces an error from command line.

Indeed, If i test the command produces ( I remove the output args) ,I got :
```console
#  ctest --test-dir D:\personal\workspace\test-sandbox\build\Debug -I 0,0,0,
Internal ctest changing into directory: D:/personal/workspace/test-sandbox/build/Debug
Test project D:/personal/workspace/test-sandbox/build/Debug
No tests were found!!!
``` 

if I remove the -I option it works : 
```console
#  ctest --test-dir D:\personal\workspace\test-sandbox\build\Debug
Internal ctest changing into directory: D:/personal/workspace/test-sandbox/build/Debug
Test project D:/personal/workspace/test-sandbox/build/Debug
    Start 1: lib.add_one
1/2 Test #1: lib.add_one ......................   Passed    0.01 sec
    Start 2: lib.failed
2/2 Test #2: lib.failed .......................***Failed    0.01 sec

50% tests passed, 1 tests failed out of 2

Total Test time (real) =   0.04 sec

The following tests FAILED:
          2 - lib.failed (Failed)
Errors while running CTest
Output from these tests are in: D:/personal/workspace/test-sandbox/build/Debug/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
```

**Comment by orjangj** [comment]
Hi. Could you have a look at my comment in #24? I think this issue is related to the fix you provided in #24. If this is the case, then we can close this issue. Removing the `--tests-information` option shouldn't be necessary as long as `ctest` is running in the right build/binary tree.

**Comment by orjangj** [comment]
FYI, this command: `ctest --test-dir D:\personal\workspace\test-sandbox\build\Debug -I 0,0,0,` won't run any tests because it doesn't specify any test numbers/indexes to run.

I'm pretty sure you would see a different result if you run: `ctest --test-dir D:\personal\workspace\test-sandbox\build\Debug -I 0,0,0,1,2` (which is the filter that `neotest-ctest` should have produced if it found runnable tests to execute).

**Comment by orjangj** [comment]
Should be fixed as part of #24 now.
